wps-cloudformation-template.yaml

```diff
@@ -56,6 +56,8 @@ Parameters:
   provenanceTemplateNonGriddedURL:
     Type: String
     Default: 'https://raw.githubusercontent.com/aodn/aws-wps/master/wps-common/src/main/resources/config/provenance_template_non_gridded.ftl'
+  sumoEndpoint:
+    Type: String
```

Comment:
This should have a default value of an empty string, so that if it is omitted, by default, it deploys the stack with no sumo integration

Reply:
Make Sense... Updated.